src/main/io/dashboard.h

```diff
@@ -16,9 +16,26 @@
  */
 
 #include "common/time.h"
+#include "config/parameter_group.h"
+#include "drivers/bus_i2c.h"
 
 #define ENABLE_DEBUG_DASHBOARD_PAGE
 
+#ifdef OLED_I2C_INSTANCE
+#define DASHBOARD_I2C_INSTANCE  OLED_I2C_INSTANCE
+#else
+#define DASHBOARD_I2C_INSTANCE  I2CDEV_1
+#endif
+
+#define DASHBOARD_I2C_ADDRESS   0x3C     // OLED at address 0x3C in 7bit
+
+typedef struct dashboardConfig_s {
+    I2CDevice device;
```

Comment:
It would be great if I2C driver can define `typedef` for bus configuration. But that won't play well with current PG implementation ...